File: Easy_problems/MergeStringAlternately_1768/Sol.py
class Solution(object):
    def mergeAlternately(self, word1, word2):
        string = ""
        
        for i in range(0, min(len(word1), len(word2)), 1):
            string = string + word1[i] + word2[i]
        
        if len(word1) > len(word2):
            string = string + word1[len(word2)::]
        else:
            string = string + word2[len(word1)::]
        
        return string

File: Easy_problems/MergeStringAlternately_1768/Sol.java
class Sol {
    public String mergeAlternately(String word1, String word2) {
        StringBuilder sb = new StringBuilder();
        char [] w1 = word1.toCharArray(), w2 = word2.toCharArray(); 
        
        for (int i = 0; i < word1.length() && i < word2.length(); i++) 
            sb.append(w1[i]).append(w2[i]);
        
        if(word1.length() > word2.length())
            sb.append(word1.subSequence(w2.length, w1.length));
        else
            sb.append(word2.subSequence(w1.length, w2.length));

        return sb.toString();
    }

}